vulnerabilities/pipes/advisory.py

```diff
@@ -27,11 +29,21 @@
 from vulnerabilities.models import Weakness
 
 
+def get_or_create_aliases(aliases: List) -> List:
```

Comment:
IMO we should have a small doctest for this.

Reply:
We have unittest for this now.